packages/cmrjs/src/cmr-utils.js

```diff
@@ -140,22 +146,32 @@ async function publishECHO10XML2CMR(cmrFile, cmrClient) {
  * @param {Object} cmrFile.metadataObject - the UMMG JSON cmr metadata
  * @param {Object} cmrFile.granuleId - the metadata's granuleId
  * @param {Object} cmrClient - a CMR instance
+ * @param {string} revisionId - Optional CMR Revision ID
  * @returns {Object} CMR's success response which includes the concept-id
  */
-async function publishUMMGJSON2CMR(cmrFile, cmrClient) {
+async function publishUMMGJSON2CMR(cmrFile, cmrClient, revisionId = undefined) {
```

Comment:
We need a unit test here to prove that the cmrRevisionId is correctly passed on to `ingestUMMGranule`

Reply:
[`4e5ded5` (#2079)](https://github.com/nasa/cumulus/pull/2079/commits/4e5ded590cd0d0e529bd197ab669e5cfe719ec61)